public/static-checkout/css/style.css

```diff
@@ -227,30 +204,14 @@ header {
   float: left;
 }
 
-.row:before,
-.row:after {
-  content:"";
-  display: table ;
-  clear:both;
-}
-
 [class*='col-'] {
   display: inline-block;
   min-height: 1px;
 }
 
-.col-1 {width: 8.33%;}
```

Comment:
I was going to use some of them for the responsiveness, but then I would need to add the class to the HTML, so I will just use width + media queries instead